types/textract/index.d.ts

```diff
@@ -12,6 +12,47 @@ export interface extractorExecOpts {
     exec: { [index: string]: string };
 }
 
+export type MimeType =
+| string
```

Comment:
I'm afraid adding `string` to this union effectively makes the whole thing *only* a `string` to TypeScript and no-one gets any value from you adding the extras. See https://github.com/microsoft/TypeScript/issues/33471

Reply:
Should I do PR again except string?

Reply:
@woskaangel you could add single utility type to create literal union, exactly like first one in this comment:
https://github.com/microsoft/TypeScript/issues/29729#issuecomment-471566609
the sole purpose is to offer both, string literals from union and optionally new value.

Reply:
> Should I do PR again except string?

I think w/o string is not good either, since textract supports `text/*` mime types, also it would be a breaking change. Since TS 4.1 there are Template literal types, which might be better suited, but they are quite new and would break compatibility. I am not exactly sure how DefinitelyTypes policies are on this matter, but IMO it isn't worth adding those mime types.